src/libvfcfuncinstrument/libVFCFuncInstrument.cpp

```diff
@@ -50,12 +50,44 @@
 #include <utility>
 #include <vector>
 
+#if LLVM_VERSION_MAJOR < 5
```

Comment:
Can we factor this common code between `libVFCFuncInstrument.cpp` and `libVFCInstrument.cpp` in a `.hpp` header ?

Reply:
We don't need these macro anymore thanks to @Avaquet who found a simpler solution